docs/src/userguide/extension_types.rst

```diff
@@ -148,8 +150,8 @@ of an extension type must be correct to access its :keyword:`cdef` attributes an
 type and does a type check instead, analogous to Pyrex's :meth:`typecheck`.
 The old behavior is always available by passing a tuple as the second parameter::
 
-    print isinstance(sh, Shrubbery)     # Check the type of sh
-    print isinstance(sh, (Shrubbery,))  # Check sh.__class__
+    print(isinstance(sh, Shrubbery))     # Check the type of sh
+    print(isinstance(sh, (Shrubbery,)))  # Check sh.__class__
```

Comment:
Interesting. I think this whole paragraph is outdated. `isinstance()` with extension types always does a type check in Cython, also when you pass a tuple. And I think I recall that even the comment on the `__class__` check no longer applies in Py3. Type checks have definitely changed since this was written.

This should be more accurate:

"To explicitly test the type of an object, use the :meth:\`isinstance\` method. For known builtin or extension types, Cython translates these into a fast and safe type check that ignores changes to the object's `__class__` attribute etc., so that after a successful `isinstance()` test, code can rely on the expected C structure of the extension type and its `cdef` attributes and methods.